bin/run

```diff
@@ -6,17 +6,14 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-const fs = require("fs");
-const path = require("path");
-const project = path.join(__dirname, "../tsconfig.json");
-const dev = fs.existsSync(project);
-
-if (dev) {
-  require("ts-node").register({ project });
-}
-
-// require(`../${dev ? "src" : "lib"}`)
-require(`../${dev ? "src" : "lib"}`)
-  .default.run()
-  // eslint-disable-next-line @typescript-eslint/no-var-requires
-  .catch(require("@oclif/errors/handle"));
+require("@oclif/command")
+  .run()
+  .then(require("@oclif/command/flush"))
+  .catch(e => {
+    if (e.message.startsWith("command")) {
+      return require("../lib/commands/lint")
+        .default.run();
```

Comment:
[BLOCKER] Not providing the ts-node register means that you must recompile on all changes before testing.  It also means I think that debugging the typescript directly will be more difficult since it will be going through the compiled javascript.

Reply:
That's actually a shortcoming in oclif. The default wrapper for a multi-command cli is ``` /tmp/mynewcli      cat bin/run        
#!/usr/bin/env node

require('@oclif/command').run()
.then(require('@oclif/command/flush'))
.catch(require('@oclif/errors/handle'))
```
They only have that dev-mode support for single command CLI's. Let me think if there's a simple way we might try to have something similar, but it's not something oclif provides support for.

Reply:
I put together a quick wrapper as bin/run-dev that gives similar function for multicommand